### ISSUE TITLE
chainCA Error

### ISSUE THREAD

**Issue by delgadillo22** [F5Networks/f5-appsvcs-extension#410]
### Environment
 * Application Services Version: Tested on AS3.24 and AS3.25
 * BIG-IP Version: 14.1.2.6 EHF

### Summary

Customer can’t get the chain CA in TLS_Client working.
Bolded below is what the customer believes has a bug.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "class": "ADC",
    "schemaVersion": "3.23.0",
    "controls": {
        "class": "Controls",
        "trace": true,
        "logLevel": "debug",
        "traceResponse": true
    },
    "tenant": {
        "class": "Tenant",
        "application": {
            "class": "Application",
            "template": "generic",
            "tls_client": {
                "class": "TLS_Client",
                "clientCertificate": "certificate"
            },
            "certificate": {
                "class": "Certificate",
                "certificate": {
                    "bigip": "/Common/default.crt"
                },
                "privateKey": {
                    "bigip": "/Common/default.key"
                },
                "chainCA": {
                    "bigip": "/Common/f5-ca-bundle.crt"
                }
            }
        }
    }
}
```

2. Observe the following cli script from the trace contains `chain none`:
>cli script __appsvcs_update {\nproc script::run {} {\nif {[catch {\ntmsh::modify ltm data-group internal __appsvcs_update records none\n} err]} {\ntmsh::create ltm data-group internal __appsvcs_update type string records none\n}\nif { [catch {\ntmsh::create auth partition tenant default-route-domain 0\ntmsh::create sys folder /tenant/application/\ntmsh::begin_transaction\ntmsh::modify auth partition tenant description \\\"Updated by AS3 at [clock format [clock seconds] -gmt true -format {%a, %d %b %Y %T %Z}]\\\"\ntmsh::create ltm profile server-ssl /tenant/application/tls_client alert-timeout indefinite allow-expired-crl disabled authenticate once authenticate-name none c3d-ca-cert none c3d-ca-key none c3d-cert-lifespan 24 c3d-cert-extension-includes \\{ basic-constraints extended-key-usage key-usage subject-alternative-name \\} cache-timeout 3600 ca-file /Common/ca-bundle.crt cert /Common/default.crt chain none ciphers DEFAULT cipher-group none crl-file none description none key /Common/default.key expire-cert-response-control drop options \\{ dont-insert-empty-fragments no-tlsv1.3 \\} peer-cert-mode ignore renegotiation enabled retain-certificate true server-name none session-ticket disabled ssl-c3d disabled ssl-forward-proxy disabled ssl-forward-proxy-bypass disabled untrusted-cert-response-control drop\ntmsh::commit_transaction\n} err] } {\ncatch { tmsh::cancel_transaction } e\nregsub -all {\"} $err {\\\"} err\ntmsh::modify ltm data-group internal __appsvcs_update records add \\{ error \\{ data \\\"$err\\\" \\} \\}\ncatch { tmsh::delete sys folder /tenant/application/ } e\ncatch { tmsh::delete auth partition tenant } e\n}}\n}

### Expected Behavior
Successful chainCA configuration

### Actual Behavior
No chainCA config.

**Comment by dstokesf5** [comment]
Added to our internal product backlog as AUTOTOOL-2217.

**Comment by dstokesf5** [comment]
This was resolved in the 3.26.0 release.
